collectd/collectd_test.go

```diff
@@ -14,22 +15,18 @@ import (
 type testServer string
 type serverResponses []serverResponse
 type serverResponse struct {
-	database, retentionPolicy, name string
-	tags                            map[string]string
-	timestamp                       time.Time
-	values                          map[string]interface{}
+	database        string
+	retentionPolicy string
+	points          []influxdb.Point
 }
 
 var responses = make(chan *serverResponse, 1024)
 
-func (testServer) WriteSeries(database, retentionPolicy, name string, tags map[string]string, timestamp time.Time, values map[string]interface{}) (uint64, error) {
+func (testServer) WriteSeries(database, retentionPolicy string, points ...influxdb.Point) (uint64, error) {
```

Comment:
Why did you decide to use varargs? It seems like if we'd pass a bunch of points in then they'd likely already be in a slice.